src/ceph-volume/ceph_volume/devices/raw/activate.py

```diff
@@ -44,8 +45,49 @@ def activate_bluestore(meta, tmpfs, systemd):
     # correctly every time
     process.run(['ln', '-snf', meta['device'], os.path.join(osd_path, 'block')])
     system.chown(os.path.join(osd_path, 'block'))
-    system.chown(osd_path)
 
+    wal_device = os.getenv("CEPH_VOLUME_OSD_WAL_DEVICE")
```

Comment:
why use environment varialbes and not --block.db etc like the prepare?

Reply:
I think we could do both, I'm just trying to not touch this code too much: https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/osd/spec.go#L71-L106

Each time we change it this will restart the OSD, so I'm trying to make a few changes as possible in it.
However, we are still in Rook's 1.3 cycle so I guess it's fine to change it.

Reply:
I think arguments are preferable over env variables by far.

Reply:
yes!